### PR TITLE
Added new consul crawler

### PR DESCRIPTION
**Pull request by liozN**
consul crawler for new jenkins plugin

### CODE REVIEW

**Comment by daniel-beck** [comment]
Please provide references to the HOSTING request and the plugin's GitHub repository in the jenkinsci organization.

**Comment by liozN** [comment]
https://github.com/jenkinsci/consul-plugin

File: consul.groovy [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!./lib/runner.groovy
+// Generates server-side metadata for Packer auto-installation
```
Comment (daniel-beck) [inline consul.groovy]: Copy-pastastic

**Comment by daniel-beck** [comment]
@liozN https://updates.jenkins-ci.org/updates/com.inneractive.jenkins.plugins.consul.ConsulInstaller.json

**Pull request merged**